Ginger/Ginger/App.xaml.cs

```diff
@@ -332,8 +341,30 @@ private bool ShouldStartGrid(string[] args)
         /// <returns>ParserResult containing parsed arguments or null.</returns>
         private ParserResult<object> ParseCommandLineArguments(string[] args)
         {
+            string[] arguments;
+            if (args.Length == 1 && System.Web.HttpUtility.UrlDecode(args[0]).Equals("ginger:///", StringComparison.OrdinalIgnoreCase))
```

Comment:
it should be "ginger://" instead of "ginger:///"

Reply:
when user want to just open the Ginger from browser at that time one "/" slash is gets added so to handle that situation we have used this condition